index.js

```diff
@@ -1,20 +1,28 @@
 'use strict';
 
-var _ = require('lodash');
 var inlineSource = require('inline-source').sync;
 
 /*
+ * Filter files to find HTML files.
  * Iterate through each file name and grab
  * the contents of each file and replace
  * with an inlined version.
  */
 
 var inline = function(options) {
   return function(files, metalsmith, done) {
-    _.forEach(files, function(file, path, files) {
-      files[path].contents = new Buffer(inlineSource(files[path].contents.toString(), options)), 'utf8';
-    });
+      var htmlFiles = Object.keys(files).filter(function(file) {
```

Comment:
just one thing here, the indentation is 4 characters, rest of the file is two. Update that then good to go, i think this being indented 4 characters has messed with the indentation down below too.